### ISSUE TITLE
Feature Request:  Ability to find which external pipeline job triggered this job

### ISSUE THREAD

**Issue by evantorrie** [screwdriver-cd/screwdriver#1139]
When a pipeline has an external pipeline trigger, e.g.

```
requires:  [~pr, ~commit, ~sd@123:main, ~sd@246:commit]
```

it should be easy to find out which of the specified external jobs (and likely the internal was the cause of this pipeline running.

While it's easy to get metadata from preceeding jobs from the same pipeline, getting metadata from other external jobs requires knowing the full external pipeline ID, i.e. `meta get example --external sd@123:main`, which means you may have to query all of them.

For purposes of writing generic scripts to pull prior pipeline metadata, it would be great to be able to do so by knowing which specific external pipeline was responsible for triggering this run of the job.

**Comment by jithin1987** [comment]
I've already captured this in https://github.com/screwdriver-cd/screwdriver/issues/922 Will prioritize this in coming sprints

**Comment by jithin1987** [comment]
Closing this in favor of #922
